src/components/GridItem/gridItemUtils.ts

```diff
@@ -0,0 +1,28 @@
+import { px } from '../../utils';
+import { GridItemProps } from './GridItem';
+
+export enum GridItemAlign {
+  center = 'center',
```

Comment:
I think props don't belong to utils file, can we create dedicated file for Props? With more advanced components we will have more props